### PR TITLE
Fix error coverage report

### PR DESCRIPTION
**Pull request by Nocccer**
- [x] adding jest.config.js for frontend
- [x] adding jest.config.js for backend

----------------------------------

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Created / Updated Tests
- [ ] Created / Updated documentation

Tested on a Clean Install with AppImage:
- [ ] Login
- [ ] Download a Game
- [ ] Import a Game
- [ ] Launch Default config
- [ ] Launch Personalized Config
- [ ] Launch wine
- [ ] Launch Proton
- [ ] Launch Custom Proton
- [ ] Launch Tools (Winetricks and Winecfg)
- [ ] Uninstall Game
- [ ] Move Game
- [ ] Languages (Test if no translation stopped working)

### CODE REVIEW

**Comment by flavioislima** [comment]
Looks good for this class but the others also must be inialialized to fix all errors.

File: electron/game_config.ts [hunk 1]
```diff
@@ -24,7 +24,24 @@ abstract class GameConfig {
 
   public abstract version : GameConfigVersion
```
Comment (dragonDScript) [inline electron/game_config.ts:25]: Declare version

**Review by adityaruplaha — APPROVED** [review]
Looks good, I'm not 100% sure we can get away with just `if (config)` after this though. I would initially set it to null and test if it was not null and only then proceed. 

But everything seems to be fine, so LGTM.

**Comment by flavioislima** [comment]
The build will never work if you use `tsconfig` like this, they need to be different.
You need to keep in mind that the electron and the src folder are two complete different worlds, its the same logic we have on a web app, being the electron the backend and the src with react the frontend. We can have similar things, types, and share a few patterns, but they are not the same, that's why we have two different `tsconfig` files.

File: electron/utils.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ import {
 import { promisify } from 'util'
 import i18next from 'i18next'
 
+import { error } from 'console'
```
Comment (dragonDScript) [inline electron/utils.ts:16]: Same here. You should use './console'

**Review by dragonDScript — CHANGES_REQUESTED** [review]
Please fix these

**Pull request closed without merging**